rules/sinks/third_parties/sdk/airflow/python.yaml

```diff
@@ -0,0 +1,53 @@
+
+#  Sink rule for ThirdParty SDK
+#  The id follows a format : "ThirdParties.SDK.<THIRD_PARTY_ORGANISATION>.<SUB_ORGANISATION_IF_APPLICABLE>"
+
+sinks:
+
+  - id: ThirdParties.Operator.Slack
+    name: Airflow Slack Operator
```

Comment:
Name these as Slack, Amazon, etc.

some of these are database operators so good to move them under Storages.

Reply:
okay sure.